perf_: Don't store community members for public chats

alexjba opened the pull request:
Avoid duplicating community members in public chats
Rationale: reduce memory footprint

<p style="margin: 0.0px 0.0px 0.0px 0.0px; font: 13.0px 'Helvetica Neue'"><br>
Startup memory footprint for status-desktop</p>

Account type | With public chat members | Without public chat members
-- | -- | --
Old account - status member | 1.30GB | 1.1GB
New account with Status | 580Mb | 550Mb
New account loading Status | 1.30 GB | 740Mb


<br class="Apple-interchange-newline">

Iterates https://github.com/status-im/status-desktop/issues/18150

alexjba committed:
perf_: Don't store community members for public chats

Iterates https://github.com/status-im/status-desktop/issues/18150

Rationale: reduce memory footprint

chore_: Isolate persistance test

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (114)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_multiplication_x: | cb78e59b | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/1/) | 2025-07-21 09:42:52 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/1//consoleText) |
| :heavy_check_mark: | cb78e59b | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6762/1/) | 2025-07-21 09:43:43 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250721-094041-cb78e5-pr6762.aar) |
| :heavy_check_mark: | cb78e59b | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6762/1/) | 2025-07-21 09:44:00 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250721-094044-cb78e5-pr6762.zip) |
| :heavy_check_mark: | cb78e59b | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6762/1/) | 2025-07-21 09:44:57 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250721-094051-cb78e5-pr6762.zip) |
| :heavy_check_mark: | cb78e59b | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6762/1/) | 2025-07-21 09:46:49 | ~6 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250721-094048-cb78e5-pr6762.zip) |
| :heavy_check_mark: | cb78e59b | [#1](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6762/1/) | 2025-07-21 09:47:29 | ~6 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250721-094055-cb78e5-pr6762.zip) |
| :heavy_multiplication_x: | cb78e59b | [#1](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/1/) | 2025-07-21 09:47:47 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/1//consoleText) |
| :heavy_check_mark: | cb78e59b | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6762/1/) | 2025-07-21 09:50:08 | ~9 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250721-094042-cb78e5-pr6762.zip) |
| :heavy_check_mark: | cb78e59b | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6762/1/) | 2025-07-21 09:50:56 | ~10 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250721-094045-cb78e5-pr6762.zip) |
| :heavy_multiplication_x: | cb78e59b | [#2](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/2/) | 2025-07-21 11:42:53 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/2//consoleText) |
| :heavy_multiplication_x: | cb78e59b | [#3](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/3/) | 2025-07-21 12:29:56 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/3//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 78661d1a | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6762/2/) | 2025-07-21 12:28:13 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250721-122533-78661d-pr6762.aar) |
| :heavy_check_mark: | 78661d1a | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6762/2/) | 2025-07-21 12:28:51 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250721-122537-78661d-pr6762.zip) |
| :heavy_check_mark: | 78661d1a | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6762/2/) | 2025-07-21 12:29:34 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250721-122543-78661d-pr6762.zip) |
| :heavy_check_mark: | 78661d1a | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6762/2/) | 2025-07-21 12:31:51 | ~6 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250721-122541-78661d-pr6762.zip) |
| :heavy_check_mark: | 78661d1a | [#2](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6762/2/) | 2025-07-21 12:33:17 | ~7 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250721-122548-78661d-pr6762.zip) |
| :heavy_check_mark: | 78661d1a | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6762/2/) | 2025-07-21 12:34:29 | ~8 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250721-122535-78661d-pr6762.zip) |
| :heavy_multiplication_x: | 78661d1a | [#4](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/4/) | 2025-07-21 12:36:21 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/4//consoleText) |
| :heavy_check_mark: | 78661d1a | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6762/2/) | 2025-07-21 12:36:28 | ~10 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250721-122538-78661d-pr6762.zip) |
| :heavy_multiplication_x: | 78661d1a | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/2/) | 2025-07-21 12:49:40 | ~23 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/2//consoleText) |
| | | | | | | |
| :heavy_check_mark: | a025db38 | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6762/3/) | 2025-07-21 12:37:13 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250721-123425-a025db-pr6762.aar) |
| :heavy_check_mark: | a025db38 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6762/3/) | 2025-07-21 12:37:27 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250721-123428-a025db-pr6762.zip) |
| :heavy_check_mark: | a025db38 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6762/3/) | 2025-07-21 12:38:29 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250721-123436-a025db-pr6762.zip) |
| :heavy_check_mark: | a025db38 | [#3](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6762/3/) | 2025-07-21 12:40:24 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250721-123440-a025db-pr6762.zip) |
| :heavy_check_mark: | a025db38 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6762/3/) | 2025-07-21 12:41:00 | ~6 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250721-123432-a025db-pr6762.zip) |
| :heavy_multiplication_x: | a025db38 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/5/) | 2025-07-21 12:42:32 | ~5 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/5//consoleText) |
| :heavy_check_mark: | a025db38 | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6762/3/) | 2025-07-21 12:43:31 | ~8 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250721-123442-a025db-pr6762.zip) |
| :heavy_check_mark: | a025db38 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6762/3/) | 2025-07-21 12:43:35 | ~9 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250721-123429-a025db-pr6762.zip) |
| :heavy_multiplication_x: | a025db38 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/3/) | 2025-07-21 13:13:53 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/3//consoleText) |
| :heavy_multiplication_x: | a025db38 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/4/) | 2025-07-21 14:06:02 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/4//consoleText) |
| | | | | | | |
| :heavy_check_mark: | cabbb9ea | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6762/4/) | 2025-07-23 11:07:46 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250723-110442-cabbb9-pr6762.aar) |
| :heavy_check_mark: | cabbb9ea | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6762/4/) | 2025-07-23 11:08:02 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250723-110445-cabbb9-pr6762.zip) |
| :heavy_check_mark: | cabbb9ea | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6762/4/) | 2025-07-23 11:08:34 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250723-110450-cabbb9-pr6762.zip) |
| :heavy_check_mark: | cabbb9ea | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6762/4/) | 2025-07-23 11:09:55 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250723-110452-cabbb9-pr6762.zip) |
| :heavy_check_mark: | cabbb9ea | [#6](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/6/) | 2025-07-23 11:11:53 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/6//consoleText) |
| :heavy_check_mark: | cabbb9ea | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6762/4/) | 2025-07-23 11:12:14 | ~7 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250723-110443-cabbb9-pr6762.zip) |
| :heavy_check_mark: | cabbb9ea | [#4](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6762/4/) | 2025-07-23 11:13:45 | ~8 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250723-110457-cabbb9-pr6762.zip) |
| :heavy_multiplication_x: | cabbb9ea | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/5/) | 2025-07-23 11:14:14 | ~9 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/5//consoleText) |
| :heavy_check_mark: | cabbb9ea | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6762/4/) | 2025-07-23 11:15:19 | ~10 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250723-110447-cabbb9-pr6762.zip) |
| | | | | | | |
| :heavy_check_mark: | 4ab172b4 | [#5](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6762/5/) | 2025-07-23 12:10:30 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250723-120806-4ab172-pr6762.aar) |
| :heavy_check_mark: | 4ab172b4 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6762/5/) | 2025-07-23 12:10:45 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250723-120809-4ab172-pr6762.zip) |
| :heavy_check_mark: | 4ab172b4 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6762/5/) | 2025-07-23 12:11:15 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250723-120813-4ab172-pr6762.zip) |
| :heavy_check_mark: | 4ab172b4 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6762/5/) | 2025-07-23 12:13:21 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250723-120817-4ab172-pr6762.zip) |
| :heavy_check_mark: | 4ab172b4 | [#5](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6762/5/) | 2025-07-23 12:13:59 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250723-120820-4ab172-pr6762.zip) |
| :heavy_check_mark: | 4ab172b4 | [#5](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6762/5/) | 2025-07-23 12:14:07 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250723-120807-4ab172-pr6762.zip) |
| :heavy_check_mark: | 4ab172b4 | [#7](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/7/) | 2025-07-23 12:15:03 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/7//consoleText) |
| :heavy_check_mark: | 4ab172b4 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6762/5/) | 2025-07-23 12:18:13 | ~10 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250723-120810-4ab172-pr6762.zip) |
| :heavy_multiplication_x: | 4ab172b4 | [#6](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/6/) | 2025-07-23 12:37:25 | ~29 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/6//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 93d4fbdc | [#6](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6762/6/) | 2025-07-24 11:49:27 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250724-114704-93d4fb-pr6762.aar) |
| :heavy_check_mark: | 93d4fbdc | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6762/6/) | 2025-07-24 11:50:04 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250724-114706-93d4fb-pr6762.zip) |
| :heavy_check_mark: | 93d4fbdc | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6762/6/) | 2025-07-24 11:50:10 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250724-114710-93d4fb-pr6762.zip) |
| :heavy_check_mark: | 93d4fbdc | [#6](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6762/6/) | 2025-07-24 11:51:35 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250724-114705-93d4fb-pr6762.zip) |
| :heavy_check_mark: | 93d4fbdc | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6762/6/) | 2025-07-24 11:52:10 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250724-114715-93d4fb-pr6762.zip) |
| :heavy_check_mark: | 93d4fbdc | [#6](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6762/6/) | 2025-07-24 11:52:57 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250724-114719-93d4fb-pr6762.zip) |
| :heavy_check_mark: | 93d4fbdc | [#8](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/8/) | 2025-07-24 11:54:01 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/8//consoleText) |
| :heavy_check_mark: | 93d4fbdc | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6762/6/) | 2025-07-24 11:56:53 | ~9 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250724-114708-93d4fb-pr6762.zip) |
| :heavy_multiplication_x: | 93d4fbdc | [#7](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/7/) | 2025-07-24 12:13:03 | ~25 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/7//consoleText) |
| | | | | | | |
| :heavy_check_mark: | e82f4d98 | [#7](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6762/7/) | 2025-07-25 11:38:16 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250725-113519-e82f4d-pr6762.aar) |
| :heavy_check_mark: | e82f4d98 | [#7](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6762/7/) | 2025-07-25 11:38:22 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250725-113525-e82f4d-pr6762.zip) |
| :heavy_check_mark: | e82f4d98 | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6762/7/) | 2025-07-25 11:38:49 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250725-113521-e82f4d-pr6762.zip) |
| :heavy_check_mark: | e82f4d98 | [#7](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6762/7/) | 2025-07-25 11:39:51 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250725-113520-e82f4d-pr6762.zip) |
| :heavy_check_mark: | e82f4d98 | [#7](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6762/7/) | 2025-07-25 11:40:27 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250725-113528-e82f4d-pr6762.zip) |
| :heavy_check_mark: | e82f4d98 | [#7](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6762/7/) | 2025-07-25 11:41:12 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250725-113533-e82f4d-pr6762.zip) |
| :heavy_check_mark: | e82f4d98 | [#9](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/9/) | 2025-07-25 11:41:47 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/9//consoleText) |
| :heavy_check_mark: | e82f4d98 | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6762/7/) | 2025-07-25 11:45:26 | ~10 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250725-113523-e82f4d-pr6762.zip) |
| :heavy_multiplication_x: | e82f4d98 | [#8](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/8/) | 2025-07-25 12:00:07 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/8//consoleText) |
| | | | | | | |
| :heavy_check_mark: | a1271e00 | [#8](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6762/8/) | 2025-07-25 13:32:49 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250725-133005-a1271e-pr6762.aar) |
| :heavy_check_mark: | a1271e00 | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6762/8/) | 2025-07-25 13:33:06 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250725-133008-a1271e-pr6762.zip) |
| :heavy_check_mark: | a1271e00 | [#8](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6762/8/) | 2025-07-25 13:35:14 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250725-133014-a1271e-pr6762.zip) |
| :heavy_check_mark: | a1271e00 | [#8](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6762/8/) | 2025-07-25 13:37:48 | ~7 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250725-133020-a1271e-pr6762.zip) |
| :heavy_check_mark: | a1271e00 | [#10](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/10/) | 2025-07-25 13:39:13 | ~8 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/10//consoleText) |
| :heavy_check_mark: | a1271e00 | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6762/8/) | 2025-07-25 13:40:41 | ~10 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250725-133009-a1271e-pr6762.zip) |
| :heavy_check_mark: | a1271e00 | [#8](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6762/8/) | 2025-07-25 13:42:19 | ~12 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250725-133006-a1271e-pr6762.zip) |
| :heavy_check_mark: | a1271e00 | [#8](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6762/8/) | 2025-07-25 13:45:14 | ~15 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250725-133012-a1271e-pr6762.zip) |
| :heavy_multiplication_x: | a1271e00 | [#9](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/9/) | 2025-07-25 13:57:44 | ~27 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/9//consoleText) |
| :heavy_check_mark: | a1271e00 | [#10](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/10/) | 2025-07-25 14:38:44 | ~25 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/10//consoleText) |
| | | | | | | |
| :heavy_check_mark: | dc5210ce | [#9](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6762/9/) | 2025-07-25 15:26:18 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250725-152330-dc5210-pr6762.aar) |
| :heavy_check_mark: | dc5210ce | [#9](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6762/9/) | 2025-07-25 15:26:23 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250725-152333-dc5210-pr6762.zip) |
| :heavy_check_mark: | dc5210ce | [#9](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6762/9/) | 2025-07-25 15:28:49 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250725-152341-dc5210-pr6762.zip) |
| :heavy_check_mark: | dc5210ce | [#9](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6762/9/) | 2025-07-25 15:29:15 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250725-152344-dc5210-pr6762.zip) |
| :heavy_check_mark: | dc5210ce | [#9](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6762/9/) | 2025-07-25 15:31:16 | ~7 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250725-152334-dc5210-pr6762.zip) |
| :heavy_multiplication_x: | dc5210ce | [#11](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/11/) | 2025-07-25 15:32:21 | ~8 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/11//consoleText) |
| :heavy_multiplication_x: | dc5210ce | [#12](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/12/) | 2025-07-25 15:43:12 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/12//consoleText) |
| :heavy_check_mark: | dc5210ce | [#11](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/11/) | 2025-07-25 15:51:51 | ~28 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/11//consoleText) |
| :heavy_check_mark: | dc5210ce | [#9](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6762/9/) | 2025-07-25 16:47:12 | ~1 hr 23 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250725-152332-dc5210-pr6762.zip) |
| :heavy_check_mark: | dc5210ce | [#9](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6762/9/) | 2025-07-25 16:50:04 | ~1 hr 26 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250725-152338-dc5210-pr6762.zip) |
| :heavy_check_mark: | dc5210ce | [#13](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/13/) | 2025-07-25 18:07:07 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/13//consoleText) |
| | | | | | | |
| :heavy_check_mark: | f2d71c24 | [#10](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6762/10/) | 2025-08-04 13:46:58 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250804-134423-f2d71c-pr6762.aar) |
| :heavy_check_mark: | f2d71c24 | [#10](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6762/10/) | 2025-08-04 13:47:26 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250804-134426-f2d71c-pr6762.zip) |
| :heavy_check_mark: | f2d71c24 | [#10](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6762/10/) | 2025-08-04 13:47:33 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250804-134431-f2d71c-pr6762.zip) |
| :heavy_check_mark: | f2d71c24 | [#10](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6762/10/) | 2025-08-04 13:48:59 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250804-134425-f2d71c-pr6762.zip) |
| :heavy_check_mark: | f2d71c24 | [#10](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6762/10/) | 2025-08-04 13:49:34 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250804-134434-f2d71c-pr6762.zip) |
| :heavy_check_mark: | f2d71c24 | [#10](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6762/10/) | 2025-08-04 13:50:36 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250804-134438-f2d71c-pr6762.zip) |
| :heavy_check_mark: | f2d71c24 | [#14](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/14/) | 2025-08-04 13:52:24 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/14//consoleText) |
| :heavy_check_mark: | f2d71c24 | [#10](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6762/10/) | 2025-08-04 13:55:09 | ~10 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250804-134428-f2d71c-pr6762.zip) |
| :heavy_multiplication_x: | f2d71c24 | [#12](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/12/) | 2025-08-04 14:10:29 | ~25 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/12//consoleText) |
| | | | | | | |
| :heavy_check_mark: | c6eb9eba | [#11](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6762/11/) | 2025-08-04 13:49:26 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250804-134705-c6eb9e-pr6762.aar) |
| :heavy_check_mark: | c6eb9eba | [#11](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6762/11/) | 2025-08-04 13:50:43 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250804-134729-c6eb9e-pr6762.zip) |
| :heavy_check_mark: | c6eb9eba | [#11](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6762/11/) | 2025-08-04 13:51:12 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250804-134739-c6eb9e-pr6762.zip) |
| :heavy_check_mark: | c6eb9eba | [#11](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6762/11/) | 2025-08-04 13:53:38 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250804-134918-c6eb9e-pr6762.zip) |
| :heavy_check_mark: | c6eb9eba | [#11](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6762/11/) | 2025-08-04 13:54:48 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250804-134939-c6eb9e-pr6762.zip) |
| :heavy_check_mark: | c6eb9eba | [#11](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6762/11/) | 2025-08-04 13:56:10 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250804-135039-c6eb9e-pr6762.zip) |
| :heavy_check_mark: | c6eb9eba | [#11](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6762/11/) | 2025-08-04 13:56:40 | ~10 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250804-134626-c6eb9e-pr6762.zip) |
| :heavy_check_mark: | c6eb9eba | [#15](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/15/) | 2025-08-04 14:00:12 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/15//consoleText) |
| :heavy_check_mark: | c6eb9eba | [#13](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/13/) | 2025-08-04 14:41:11 | ~30 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/13//consoleText) |
| | | | | | | |
| :heavy_check_mark: | a8d58e25 | [#12](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6762/12/) | 2025-08-05 13:21:26 | ~2 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250805-131830-a8d58e-pr6762.zip) |
| :heavy_check_mark: | a8d58e25 | [#12](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6762/12/) | 2025-08-05 13:21:34 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250805-131821-a8d58e-pr6762.aar) |
| :heavy_check_mark: | a8d58e25 | [#12](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6762/12/) | 2025-08-05 13:21:41 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250805-131824-a8d58e-pr6762.zip) |
| :heavy_check_mark: | a8d58e25 | [#12](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6762/12/) | 2025-08-05 13:22:57 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250805-131822-a8d58e-pr6762.zip) |
| :heavy_check_mark: | a8d58e25 | [#12](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6762/12/) | 2025-08-05 13:24:11 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250805-131837-a8d58e-pr6762.zip) |
| :heavy_check_mark: | a8d58e25 | [#12](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6762/12/) | 2025-08-05 13:24:23 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250805-131833-a8d58e-pr6762.zip) |
| :heavy_check_mark: | a8d58e25 | [#16](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/16/) | 2025-08-05 13:25:36 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/16//consoleText) |
| :heavy_check_mark: | a8d58e25 | [#12](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6762/12/) | 2025-08-05 13:28:26 | ~9 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250805-131827-a8d58e-pr6762.zip) |
| :heavy_check_mark: | a8d58e25 | [#14](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/14/) | 2025-08-05 13:43:30 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/14//consoleText) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | b7ae389f | [#13](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6762/13/) | 2025-08-06 11:28:34 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250806-112608-b7ae38-pr6762.aar) |
| :heavy_check_mark: | b7ae389f | [#13](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6762/13/) | 2025-08-06 11:29:06 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250806-112612-b7ae38-pr6762.zip) |
| :heavy_check_mark: | b7ae389f | [#13](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6762/13/) | 2025-08-06 11:29:36 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250806-112616-b7ae38-pr6762.zip) |
| :heavy_check_mark: | b7ae389f | [#13](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6762/13/) | 2025-08-06 11:30:40 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250806-112610-b7ae38-pr6762.zip) |
| :heavy_check_mark: | b7ae389f | [#13](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6762/13/) | 2025-08-06 11:31:30 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250806-112620-b7ae38-pr6762.zip) |
| :heavy_check_mark: | b7ae389f | [#13](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6762/13/) | 2025-08-06 11:32:06 | ~5 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250806-112623-b7ae38-pr6762.zip) |
| :heavy_check_mark: | b7ae389f | [#17](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/17/) | 2025-08-06 11:33:38 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/17//consoleText) |
| :heavy_check_mark: | b7ae389f | [#13](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6762/13/) | 2025-08-06 11:37:01 | ~10 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250806-112613-b7ae38-pr6762.zip) |
| :heavy_check_mark: | b7ae389f | [#15](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/15/) | 2025-08-06 11:51:26 | ~25 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/15//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 682a3588 | [#14](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6762/14/) | 2025-08-07 06:58:36 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250807-065556-682a35-pr6762.aar) |
| :heavy_check_mark: | 682a3588 | [#14](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6762/14/) | 2025-08-07 06:59:00 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250807-065559-682a35-pr6762.zip) |
| :heavy_check_mark: | 682a3588 | [#14](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6762/14/) | 2025-08-07 06:59:54 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250807-065602-682a35-pr6762.zip) |
| :heavy_check_mark: | 682a3588 | [#14](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6762/14/) | 2025-08-07 07:00:47 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250807-065557-682a35-pr6762.zip) |
| :heavy_check_mark: | 682a3588 | [#14](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6762/14/) | 2025-08-07 07:01:19 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250807-065606-682a35-pr6762.zip) |
| :heavy_check_mark: | 682a3588 | [#14](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6762/14/) | 2025-08-07 07:02:18 | ~6 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250807-065611-682a35-pr6762.zip) |
| :heavy_check_mark: | 682a3588 | [#18](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/18/) | 2025-08-07 07:03:20 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6762/18//consoleText) |
| :heavy_check_mark: | 682a3588 | [#14](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6762/14/) | 2025-08-07 07:05:53 | ~9 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250807-065600-682a35-pr6762.zip) |
| :heavy_check_mark: | 682a3588 | [#16](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/16/) | 2025-08-07 07:20:37 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6762/16//consoleText) |

jrainville commented on protocol/communities/community.go:
```diff
@@ -2603,9 +2549,13 @@ func (o *Community) createChat(chatID string, chat *protobuf.CommunityChat) erro
+	if o.channelEncrypted(chatID) {
+		for pubKey, member := range o.config.CommunityDescription.Members {
+			chat.Members[pubKey] = member
+		}
```
This function is called when we first call the creation. So if it is encrypted, the evaluation is not done yet. Then, we can just leave the members empty I think. It will be populated later.

jrainville reviewed (commented):
Looks good. 

I think you could add a test or update an existing one to confirm that only private channels have a member list

alexjba commented:
ohh, it's so not ready. Moving to draft

alexjba commented on protocol/communities_messenger_token_permissions_test.go:
```diff
@@ -1736,7 +1768,7 @@ func (s *MessengerCommunitiesTokenPermissionsSuite) TestResendEncryptionKeyOnBac
 	waitOnChannelKeyToBeDistributedToBob := s.waitOnKeyDistribution(func(sub *CommunityAndKeyActions) bool {
 		action, ok := sub.keyActions.ChannelKeysActions[chat.CommunityChatID()]
-		if !ok || action.ActionType != communities.EncryptionKeyAdd {
+		if !ok || action.ActionType != communities.EncryptionKeySendToMembers {
```
can't really tell why this side-effect. But The members are sent on `EncryptionKeySendToMembers`, not on `EncryptionKeyAdd`

osmaczko replied:
```diff
diff --git a/protocol/communities/community_encryption_key_action.go b/protocol/communities/community_encryption_key_action.go
index 961c80d71..17cd5e6ac 100644
--- a/protocol/communities/community_encryption_key_action.go
+++ b/protocol/communities/community_encryption_key_action.go
@@ -86,11 +86,17 @@ func evaluateChannelLevelEncryptionKeyActions(origin, modified *Community, chang
             membersRemoved = chatChanges.MembersRemoved
         }
 
+        // Channel with 0 members is considered open
+        allMembers := modified.config.CommunityDescription.Chats[channelID].Members
+        if len(allMembers) == 0 {
+            allMembers = modified.config.CommunityDescription.Members
+        }
+
         result[channelID] = *evaluateEncryptionKeyAction(
             origin.ChannelEncrypted(channelID),
             modified.ChannelEncrypted(channelID),
             changes.ControlNodeChanged != nil,
-            modified.config.CommunityDescription.Chats[channelID].Members,
+            allMembers,
             membersAdded,
             membersRemoved,
         )

```

jrainville reviewed (approved):
Code looks good! We should test it thoroughly in the app before merging

alexjba commented:
> Code looks good! We should test it thoroughly in the app before merging

Might be worth trying an e2e on desktop as well

igor-sirotin commented:
@alexjba how did you test it? By joining Status community or a specifically created one? With how many members then?

I'm working on automated performance tests, wanna try to track the improvement with it

alexjba commented:
> @alexjba how did you test it? By joining Status community or a specifically created one? With how many members then?
> 
> I'm working on automated performance tests, wanna try to track the improvement with it

All my testes have been done around Status community. Either loading Status as spectator or loading Status as member

igor-sirotin commented:
FYI, I decided not to try to implement this test as a benchmark, as it's too complex. 
For such tests we should rely on Vac DST. If needed, they can implement and run it for us, and simulate thousands of users in a community.

alexjba commented:
> FYI, I decided not to try to implement this test as a benchmark, as it's too complex. 
> 
> For such tests we should rely on Vac DST. If needed, they can implement and run it for us, and simulate thousands of users in a community.

That's awesome! Would be great if we could get this kind of tools available for status-desktop. At least the simulation part. Maybe we could get some nighty tests running on big communities

osmaczko commented on protocol/communities/community.go:
```diff
@@ -2154,9 +2172,19 @@ func (o *Community) CanPost(pk *ecdsa.PublicKey, chatID string, messageType prot
 		return false, nil
 	}
 
+	if o.IsPrivilegedMember(pk) || common.IsPubKeyEqual(pk, o.ControlNode()) {
+		// If the user is a privileged member, they can post in any chat
+		return true, nil
+	}
```
Seems to be already covered by `o.CanView`.

alexjba replied:
That's true, but I think we need this short-circuit in both.

osmaczko replied:
`CanView` is literally one line above :thinking:

alexjba replied:
Maybe I'm missing your point.

We have 'canView' with a short-circuit for privileged member. Then we have 'canPost' -> 'canView' + same short-circuit for privileged members.

Both use the same check for privileged members, true. Now if 'canView' returns true it doesn't necessarily mean the user is privileged or he can post.

osmaczko replied:
I've misread the logic :facepalm: You are right, sorry for confusion.

osmaczko commented on protocol/communities/manager.go:
```diff
@@ -1320,7 +1320,6 @@ func (m *Manager) reevaluateMemberChannelsPermissions(community *Community, memb
 
 		// ensure member is added if channel has no permissions
```
Comment needs update.

osmaczko commented on protocol/persistence_test.go:
```diff
@@ -1670,21 +1670,37 @@ func TestSaveCommunityChat(t *testing.T) {
-	communityChat := &protobuf.CommunityChat{
-		Identity:    identity,
-		Permissions: permissions,
+	chat := &Chat{
```
Not sure I understand the need of this change.

alexjba replied:
ohh, looks like I've squashed the commits for some reason and now I can't remember why I've isolated this one. I've probably had some dependency issues after changing `CreateCommunityChat`

alexjba replied:
will look into it

alexjba replied:
ok, I remember now. With the changes in `CreateCommunityChat` we need a community to create a chat - Not sure how to easily create a valid one. Sice this test is meant to test the save to persistency and I've just removed the `CreateCommunityChat` dependency.

osmaczko commented on protocol/chat.go:
```diff
@@ -557,6 +557,16 @@ func CreateCommunityChat(orgID, chatID string, orgChat *protobuf.CommunityChat,
+	// No need for members in public channels
+	if !org.ChannelEncrypted(chatID) {
+		chat.Members = []ChatMember{}
+	}
```
Maybe parametrize `createCommunityChat` with `populateMembers` boolean instead? It will save some allocations.

alexjba replied:
good point! Looking into it

osmaczko reviewed (commented):
Looks pretty good!

osmaczko commented on protocol/communities_messenger_token_permissions_test.go:
```diff
@@ -997,23 +1004,24 @@ func (s *MessengerCommunitiesTokenPermissionsSuite) TestAnnouncementsChannelPerm
 			}
-			member := channel.Members[s.bob.IdentityPublicKeyString()]
-			return member != nil && member.ChannelRole == protobuf.CommunityMember_CHANNEL_ROLE_VIEWER
+			return c.IsMemberLikelyInChat(chat.CommunityChatID())
```
```diff
diff --git a/protocol/communities_messenger_token_permissions_test.go b/protocol/communities_messenger_token_permissions_test.go
index 198138994..5b18c071a 100644
--- a/protocol/communities_messenger_token_permissions_test.go
+++ b/protocol/communities_messenger_token_permissions_test.go
@@ -1017,18 +1017,13 @@ func (s *MessengerCommunitiesTokenPermissionsSuite) TestAnnouncementsChannelPerm
             if channel == nil || len(channel.Members) != 0 {
                 return false
             }
-            return c.IsMemberLikelyInChat(chat.CommunityChatID())
+            return c.IsMemberInChat(s.bob.IdentityPublicKey(), chat.CommunityChatID()) &&
+                c.IsMemberLikelyInChat(chat.CommunityChatID())
         },
         "no community that satisfies criteria",
     )
-
     s.Require().NoError(err)
 
-    // bob should be in the bloom filter list
-    community, err = s.bob.communitiesManager.GetByID(community.ID())
-    s.Require().NoError(err)
-    s.Require().True(community.IsMemberLikelyInChat(chat.CommunityChatID()))
-
     // bob can't post
     msg := &common.Message{
         ChatMessage: &protobuf.ChatMessage{

```

osmaczko commented on protocol/communities_messenger_token_permissions_test.go:
```diff
@@ -2015,8 +2165,7 @@ func (s *MessengerCommunitiesTokenPermissionsSuite) TestImportDecryptedArchiveMe
 		}
-		_, ok = action.Members[common.PubkeyToHex(&s.owner.identity.PublicKey)]
-		return ok
+		return true
```
?

alexjba replied:
should not be needed with this change below 🤞 It's the same issue with missing members on `communities.EncryptionKeyAdd`

https://github.com/status-im/status-go/pull/6762/files/c6eb9ebaaabd1615413183f27ced52c1706a7ebd#r2253508272

osmaczko commented on protocol/communities_messenger_token_permissions_test.go:
```diff
@@ -1760,8 +1905,13 @@ func (s *MessengerCommunitiesTokenPermissionsSuite) TestResendEncryptionKeyOnBac
+
 			channel := c.Chats()[chat.CommunityChatID()]
-			if channel != nil && len(channel.Members) < 2 {
+			if channel != nil && len(channel.Members) == 0 {
```
Please adapt comment.
> 	// channel members should not be empty,

osmaczko commented on protocol/communities/community.go:
```diff
@@ -1690,17 +1697,18 @@ func dehydrateChannelsMembers(description *protobuf.CommunityDescription) {
+					membersToKeep[id] = member
+				}
 			}
+			channel.Members = membersToKeep // keep only members that can post
```
Does the status-desktop client need to adapt to this concept, i.e. only posters on announcement channels? If so, this would be a breaking protocol change, as it goes over the wire and will eventually reach older clients.

alexjba replied:
Luckily no. We're using the community members for chats that are not encrypted

https://github.com/status-im/status-desktop/blob/master/src/app_service/service/chat/dto/chat.nim#L326-L327

osmaczko replied:
Is `tokenGated == encrypted`?

alexjba replied:
seems so

https://github.com/status-im/status-go/blob/develop/protocol/communities/community.go#L228

osmaczko replied:
> seems so
> 
> https://github.com/status-im/status-go/blob/develop/protocol/communities/community.go#L228

Confusing naming :sweat_smile: 

Please make sure announcement channels work, also for custom members (not only privileged ones) being posters :pray:

alexjba committed:
fix_: Fixing the key propagation for public chats

+ small changes on comments, tests and `CreateCommunityChat`

alexjba commented:
@jrainville @osmaczko Found an issue with new control node + old clients. The old clients will deduce the view permissions based on `func (o *Community) CanView(pk *ecdsa.PublicKey, chatID string) bool`. And once the user is a member it will always be false because the `func hydrateChannelsMembers(description *protobuf.CommunityDescription)` function will skip filling the community members when the chat has any permissions. As a result we'll always get this view.

<img width="1120" height="598" alt="Screenshot 2025-08-06 at 10 48 45" src="https://github.com/user-attachments/assets/e66799c4-351f-4f1b-9cf2-03122d78df8d" />

alexjba commented:
testing now with new control node + old mobile app. But I assume it's the same

alexjba commented:
Old mobile clients are fine. The clients can see the messages and the full members list. Looks like it's using a different mechanism

![IMG_D016C603E614-1](https://github.com/user-attachments/assets/f14ac7f8-5951-4052-8144-e332982db678)

alexjba commented:
ok, the easy way out would probably be to add all member to the announcements channels

alexjba commented:
This last commit is fixing it. When publishing the community we're adding the viewers to announcement channels. When receiving a community we're dehydrating the viewers from announcement channels.

https://github.com/status-im/status-go/pull/6762/commits/b7ae389fd0ca5ec6ae53e284cef6b5e16ce395de

osmaczko reviewed (commented):
> This last commit is fixing it. When publishing the community we're adding the viewers to announcement channels. When receiving a community we're dehydrating the viewers from announcement channels.

I’m in favor of keeping it simple and going back to all members with roles for announcement channels. Current solution doesn’t offer much in terms of memory or network bandwidth savings, but it does introduce complexity and makes the code harder to reason about.

alexjba committed:
fix_: Announcement channels hold all the members list

Announcement channels - identified by `no encryption` AND `has post permissions` will hold the entire members list.

alexjba commented:
@osmaczko I've updated the Announcement channels to hold all members - replaced the last commit with a new one. Please have a look 🙏 

The memory impact is really minimal
